server/src/scoring/handlers/userHandlers.ts

```diff
@@ -0,0 +1,27 @@
+﻿import { RequestHandler } from 'express';
+
+import { prisma } from '~/common/prisma';
+
+import { validateUser } from './../validations/userValidate';
+
+export const getUserMandatoryInfo: RequestHandler = async (req, res) => {
+  const { error } = validateUser(req.body);
+
+  if (error) return res.status(400).send(error.details[0]?.message);
+
+  if (!req.user) return res.status(401).send('brak autoryzacji');
```

Comment:
Tu nie ma opcji by req.user nie istniał, więc to sprawdzenie jest zbędne

Reply:
typescript podpowiada co innego ale dalem wszedzie '?' i usunalem if-a